src/GoogleSheets/packages/schedule/Schedule/Timesheet.go

```diff
@@ -155,6 +155,7 @@ func (t *timesheet) getShifts(employeeId string, schedule *allSchedules, reverse
 				StartTime:     schedule.Shifts[i][1].(string),
 				EndTime:       schedule.Shifts[i][2].(string),
 				BreakDuration: schedule.Shifts[i][3].(string),
+				NetHours:      schedule.Shifts[i][4].(string),
```

Comment:
Let's make this a float instead of a string (https://yourbasic.org/golang/convert-string-to-float/). Later, we can think of some "net hours summary" features.